clover-core/src/test/groovy/com/atlassian/clover/JavaSyntax115CompilationTest.groovy

```diff
@@ -0,0 +1,34 @@
+package com.atlassian.clover
+
+import com.atlassian.clover.util.FileUtils
+import org.apache.tools.ant.util.JavaEnvUtils
```

Comment:
This import seems to be unused. You can replace it with a qualified import of `clover..JavaEnvUtils`.

Reply:
Thank you for pointing this out. I have fixed it.